.github/workflows/tests.yml

```diff
@@ -0,0 +1,52 @@
+# This workflow will install Python dependencies, run tests with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: commcare-cloud tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+
+    runs-on: ubuntu-18.04
+
+    env:
+      ANSIBLE_ROLES_PATH: "~/.ansible/roles"
+      CI_TEST: 1
+      TEST: main
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python 3.6
+      uses: actions/setup-python@v3
+      with:
+        python-version: "3.6"
+
+    - name: Install dependencies
+      run: |
+        sudo apt-get clean
+        sudo apt-get update -qq
```

Comment:
What's the purpose of these two operations? I don't see any package being installed via apt, which could benefit from a prior `update`, and `clean` is normally used to "clear out the local repository of retrieved package files", freeing up disk space after a package installation.

Reply:
I think again this is for the later ansible run that targets `local`. If my plan of moving to a container works out I think we could get rid of this.

This too was copied from `.travis.yml`